lib/auth/grpcserver.go

```diff
@@ -2707,9 +2707,8 @@ func (g *GRPCServer) StreamSessionEvents(req *proto.StreamSessionEventsRequest,
 			if err := stream.Send(oneOf); err != nil {
 				return trail.ToGRPC(trace.Wrap(err))
 			}
-		case <-ctx.Done():
-			close(c)
-			return trail.ToGRPC(trace.Wrap(ctx.Err()))
+		case err := <-e:
+			return trail.ToGRPC(trace.Wrap(err))
```

Comment:
if `e` is closed this will just return `nil` right?

Reply:
From my understanding single value receive select statements on a closed channel never happen. i.e that path is never taken

Reply:
Update: I was wrong, will update to never close e.